examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-bed.prm

```diff
@@ -0,0 +1,298 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 3
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method               = bdf1
+  set number mesh adapt    = 0
+  set output name          = result
+  set output frequency     = 10
+  set startup time scaling = 0.6
+  set time end             = 5
+  set time step            = 0.001
+  set subdivision          = 1
+  set log precision        = 10
+  set output path          = ./output/
+end
```

Comment:
```suggestion
subsection simulation control
  set method               = bdf1
  set output name          = result
  set output frequency     = 10
  set startup time scaling = 0.6
  set time end             = 5
  set time step            = 0.001
  set output path          = output/
end
```

Higher value of subdivision should only be used if you are going to higher order.